COVIDWatch iOS/ViewControllers/Home.swift

```diff
@@ -17,13 +17,51 @@ class Home: BaseViewController {
     var testedButton = Button(text: "Tested for COVID-19?",
                               subtext: "Share your result anonymously to help keep your community stay safe.")
     var infoBanner = InfoBanner(text: "You may have been in contact with COVID-19")
+    var notificationsObserver: NSObjectProtocol?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // prevent removal of permissions by accident after allowing them
+        self.notificationsObserver = NotificationCenter.default.addObserver(
+            forName: UIApplication.didBecomeActiveNotification,
+            object: nil, queue: OperationQueue.main
+        ) { [weak self] _ in
+            self?.forceNotificationsEnabled()
+        }
+        self.forceNotificationsEnabled()
+    }
     override func viewDidLayoutSubviews() {
         super.viewDidLayoutSubviews()
         drawScreen()
     }
     @objc func test() {
          performSegue(withIdentifier: "test", sender: self)
     }
+    @objc func forceNotificationsEnabled() {
+        UNUserNotificationCenter.current().getNotificationSettings { (settings) in
+            guard settings.authorizationStatus != .authorized else { return }
+            let notificationsSettingsAlert = UIAlertController(
+                title: NSLocalizedString("Notifications Required", comment: ""),
+                message: "Please turn on Notifications in Settings", preferredStyle: .alert
+            )
+            notificationsSettingsAlert.addAction(
+                UIAlertAction(
+                    title: NSLocalizedString("Open Settings", comment: ""),
+                    style: .default,
+                    handler: { _ in
+                        if let url = URL(string: UIApplication.openSettingsURLString) {
+                            UIApplication.shared.open(url)
+                        }
+                    }
+                )
+            )
+            DispatchQueue.main.async {
+                self.present(notificationsSettingsAlert, animated: true)
```

Comment:
We may want to revisit this later if this is expected to require users to enable it before they continue.  

If there are other view controllers presented after this one, this may fail to show since its not the "top" view controller.

Reply:
Hmm ok